maintenance/README

```diff
@@ -12,16 +12,16 @@ to the root of your MediaWiki installation first. This is also required if
 you use a symbolic link from ./extensions/SemanticMediaWiki to the actual
 installation directory of SMW. Setting environment variables is different
 for different operating systems and shells, but can normally be done from
-the command line right before the php call. On Bash(Linux), e.g., one can
-use the following call to execute SMW_setup.php with different MW location.
+the command line right before the php call. On Bash (Linux), e.g. one can
+use the following call to execute setupStore.php with a different MW
+location:
 
-  export MW_INSTALL_PATH="/path/to/mediawiki" &&  php SMW_setup.php
+  export MW_INSTALL_PATH="/path/to/mediawiki" && php setupStore.php
 
 In some setups that use a lot of shared code for many wikis, it might be
-required to specify the location of LocalSettings.php and AdminSettings.php
-explicitly, too:
+required to specify the location of LocalSettings.php explicitly, too:
 
   export MW_INSTALL_PATH="/path/to/mediawiki" &&
-  php SMW_setup.php --conf /path/to/mediawiki/LocalSettings.php --aconf /path/to/mediawiki/AdminSettings.php
```

Comment:
Indeed it is. "AdminSettings.php" was removed in MediaWiki 1.17 so we are actually talking about 2010. :)